docs/_plugins/layer-switching-controls/l-basemapcontrol.md

```diff
@@ -0,0 +1,12 @@
+---
+name: L.basemapControl
+category: layer-switching-controls
+repo: https://github.com/Urban96/L.basemapControl
+author: Daniel Urban
+author-url: https://github.com/Urban96
+demo: https://urban96.github.io/L.basemapControl/examples/example1.html
+compatible-v0:
+compatible-v1: true
+---
+
+Basemap Control with Synchronized Minimap
```

Comment:
Please change this describtion

Reply:
Done. I hope this description better captures what the plugin does.